src/components/navigation/navigation.dom.js

```diff
@@ -31,6 +38,18 @@ domready(async () => {
         new searchToggle(toggleSearchBtn, searchEl, searchHideClass).registerEvents();
     }
 
+    if (toggleHeaderSearchBtn) {
+        const searchToggle = (await import('./navigation')).default;
+
+        new searchToggle(toggleHeaderSearchBtn, headerSearchEl, headerSearchHideClass, openIcon, closeIcon).registerEvents();
+    }
+
+    if (toggleNavMenuBtn) {
```

Comment:
Shouldn't this be `toggleHeaderMenuBtn` to match the search one?

Same with the toggle vars